app/resources/v1/printer_resource.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module V1
+  # PrinterResource
+  class PrinterResource < JSONAPI::Resource
+    model_name 'Printer', add_model_hint: false
+
+    attributes :name, :labware_type, :active?, :created_at, :updated_at, :deactivated_at
+
+    paginator :paged
+
+    filter :name, apply: lambda { |records, value, _options|
+      records.where('name LIKE ?', "%#{value[0]}%")
+    }
+
+    filter :labware_type, apply: lambda { |records, value, _options|
+      labware_types = Printer.labware_types.keys
+      unless labware_types.include?(value[0])
+        raise(JSONAPI::Exceptions::InvalidFilterValue.new(:labware_type, value[0]))
+      end
+
+      records.where(labware_type: value[0])
+    }
+
+    filter :active, apply: lambda { |records, value, _options|
```

Comment:
Since you have a scope on the model `filter :active` might just work on its own without a custom apply.

Reply:
Ignore  ^ I realised it should return inactive if false.